openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -147,12 +147,16 @@
 - control_key: IA-2 (12)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-317
+        OCP offloads (via: https://access.redhat.com/documentation/en-us/
+        openshift_container_platform/4.3/html/authentication/
+        understanding-identity-provider)
+        responsibility for authentication, session management, and OAuth
+        controls to the hosting environment’s LDAP and leverages CAC/PKI,
+        FIPS 140-2 compliant ciphers, SSO and does not provide for group
+        accounts.
```

Comment:
```suggestion
        Acceptance and electronic verification of Personal
        Identity Verification (PIV) credentials is the responsibility
        of the organizational identity provider (such as
        LDAP, Active Directory, Red Hat SSO, etc).

        To aid system administrators, documentation on
        supported authentication providers can be found in the
        OpenShift product documentation:
        https://access.redhat.com/documentation/en-us/openshift_container_platform/latest/html/authentication/understanding-identity-provider

```

Reply:
I like the additional context

Reply:
Ok, merging this suggestion into the PR. What about flipping the implementation_status to ``not applicable``?